packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -246,11 +236,6 @@ export type AutocompleteProps<
 > &
   Pick<HtmlProps, "ariaDescribedBy" | "testId" | "translate">;
 
-const ListboxContainer = styled.div`
-  width: 100%;
-  height: 100%;
-`;
-
```

Comment:
Huh, why was this here?

Reply:
This was for virtualization. Got moved to the hook